tests/integration/cmd_dev_test.go

```diff
@@ -3209,4 +3209,24 @@ CMD ["npm", "start"]
 			}))
 		}
 	}
+	Context("Devfile contains pod-overrides and container-overrides attributes", func() {
+		const (
+			// hard coded from Devfile
+			podServiceAccountName = "new-service-account"
+		)
+		BeforeEach(func() {
+			helper.CopyExample(filepath.Join("source", "nodejs"), commonVar.Context)
+			helper.CopyExampleDevFile(filepath.Join("source", "devfiles", "nodejs", "devfile-pod-container-overrides.yaml"), filepath.Join(commonVar.Context, "devfile.yaml"), helper.DevfileMetadataNameSetter(cmpName))
+		})
+		It("should override the content in the pod it creates for the component on the cluster", func() {
```

Comment:
Just curious: can we use the same test on Podman as well? Or if it does not work, maybe we can try to override some specific fields recognized by Podman..